index.js

```diff
@@ -28,8 +28,16 @@ function start(clusterProcessId = 0) {
   .then(() => subscriberInit.init(io, processName));
 }
 
+function startWithKafkaLogging() {
```

Comment:
this function name doesn't seem right - the decision about whether to use kafka logging or not is embedded in the function implementation itself, right?

Reply:
Makes sense. Fixed.